protos/bottle/fulfillment/v1/line_group.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package bottle.fulfillment.v1;
+
+import "bottle/catalog/v1/product.proto";
+
+message LineGroup {
+  string id = 1;
+  string name = 2;
+  int32 quantity = 3;
+
+  bottle.catalog.v1.Product catalog_product = 4;
+
+  enum LineGroupCategory {
+    LINE_GROUP_CATEGORY_UNSPECIFIED = 0;
+    LINE_GROUP_CATEGORY_DESKTOP = 1;
+    LINE_GROUP_CATEGORY_LAPTOP = 2;
+    LINE_GROUP_CATEGORY_SERVER = 3;
+    LINE_GROUP_CATEGORY_APPAREL = 4;
+  }
+  LineGroupCategory category = 5;
```

Comment:
Right now we only really need `computer` and `apparel` but this is going to grow down the line so I figured I'd just start with what I know we will need in the future

Reply:
Wouldn't this live on the product itself?